assets/section-multicolumn.css

```diff
@@ -36,13 +36,15 @@
   margin-right: auto;
 }
 
-.multicolumn .button {
+.multicolumn .button,
+.multicolumn .link {
   margin-top: 1.5rem;
```

Comment:
I was using 1rem on my [collection list PR](https://github.com/Shopify/dawn/pull/1491). We should try and agree on a value 🤔